sos/plugins/juju_controller.py

```diff
@@ -0,0 +1,157 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+# Copyright (C) 2019 Nick Niehoff <nick.niehoff@canonical.com>
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import re
+import os
+from sos.plugins import Plugin, UbuntuPlugin
+
+
+# A Juju Controller is a juju machine so this plugin assumes the juju_machine
+# plugin will be executed to collect that information
+class JujuController(Plugin, UbuntuPlugin):
+    """ Juju orchestration tool - Controller
+    """
+
+    plugin_name = "juju_controller"
+    profiles = ("virt", "sysmgmt")
+
+    # Use files instead of packages here because there is no package installed
+    #  on a juju controller that uniquely identifies it:
+    files = "/var/lib/juju/db"
+
+    option_list = [
+        (
+            "export-mongodb",
+            "Export mongodb collections as json files",
+            "",
+            False,
+        )
+    ]
```

Comment:
Stylistic nitpick: no need to space this out so much, it'd be easier to read at a glance if it were all together.

```python
option_list = [
    ('export-mongodb', 'Export mongodb collections as json', 'fast', False)
]
```

Reply:
Fixed on 9c909bdb8632d8bc15671235eb125d4268e0218b - Had to put on two lines or it would exceed 79 chars